apps/neuvector/neuvector/aat/00.yaml

```diff
@@ -32,7 +32,12 @@ spec:
           secretName: nv-storage-secret
           shareName: neuvector-data-00
         ingress:
-          host: neuvector00.service.core-compute-aat.internal
+          enabled: true
```

Comment:
We don't need ingress for api here, its needed only on 01 for a specific use case.

Reply:
removed